src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -422,86 +441,61 @@ interface IOptionSettlementEngine {
      * @param claimId The ID of the claim to redeem.
      */
     function redeem(uint256 claimId) external;
 
     /*//////////////////////////////////////////////////////////////
-    //  Protocol Admin
+    //  Exercise Options
     //////////////////////////////////////////////////////////////*/
 
     /**
-     * @notice Check if the protocol fee switch is enabled.
-     * @return Whether or not the protocol fee switch is enabled.
-     */
-    function feeSwitch() external view returns (bool);
-
-    /**
-     * @notice Sets the protocol fee on / off.
-     * @param enabled Whether or not the protocol fee switch should be enabled.
-     */
-    function setFeeSwitch(bool enabled) external;
-
-    /**
-     * @notice The protocol fee, expressed in basis points.
-     * @return The fee in basis points.
+     * @notice Exercises specified amount of optionId, transferring in the exercise asset,
+     * and transferring out the underlying asset if requirements are met.
+     * @param optionId The option token id of the option type to exercise.
+     * @param amount The amount of option contracts to exercise.
      */
-    function feeBps() external view returns (uint8);
+    function exercise(uint256 optionId, uint112 amount) external;
 
-    /**
-     * @notice The balance of protocol fees for a given token which have not yet
-     * been swept.
-     * @param token The token for the unswept fee balance.
-     * @return The balance of unswept fees.
-     */
-    function feeBalance(address token) external view returns (uint256);
+    /*//////////////////////////////////////////////////////////////
+    //  Privileged Functions
+    //////////////////////////////////////////////////////////////*/
 
     /**
-     * @notice Returns the address to which protocol fees are swept.
-     * @return The address to which fees are swept.
+     * @notice Enables or disables protocol fees.
+     * @param enabled Whether or not protocol fees should be enabled.
      */
-    function feeTo() external view returns (address);
+    function protocolFees(bool enabled) external;
 
     /**
-     * @notice Updates the address fees can be swept to.
-     * @param newFeeTo The new address to which fees will be swept.
+     * @notice Updates the address fees are swept to.
+     * @param newFeeTo The new address which fees are swept to.
      */
     function setFeeTo(address newFeeTo) external;
 
     /**
-     * @notice Sweeps fees to the feeTo address if there are more than 0 wei for
-     * each address in tokens.
-     * @param tokens The tokens for which fees will be swept to the feeTo address.
+     * @notice Updates the contract address for generating token URIs for tokens.
+     * @param newTokenURIGenerator The address of the new ITokenURIGenerator contract.
      */
-    function sweepFees(address[] memory tokens) external;
+    function setTokenURIGenerator(address newTokenURIGenerator) external;
```

Comment:
Move sweepFees out of Privileged Functions section, yeah?

Otherwise re-order to the top maybe? Bottom works too.